logbook-core/src/main/java/org/zalando/logbook/QueryParameters.java

```diff
@@ -111,16 +246,15 @@ public static QueryParameters parse(@Nullable final String queryString) {
             return EMPTY;
         }
 
-        final List<String> entries = Splitter.on("&").splitToList(queryString);
-        final Multimap<String, String> parameters = LinkedHashMultimap.create(entries.size(), 1);
-        final Splitter splitter = Splitter.on('=');
+        final List<String> entries = Arrays.asList(queryString.split("&"));//Splitter.on("&").splitToList(queryString);
```

Comment:
There is a separate [issue for improving query parameter][1] handling, let's not change this in this pr.

 [1]: https://github.com/zalando/logbook/issues/50